versions/blockspacerace_versions.js

```diff
@@ -1,9 +1,9 @@
 const blockspaceraceVersions = Object.freeze({
   "app-latest-tag": "v0.12.0",
-  "app-latest-sha": "67c90ede03d2543ce9b02726dfe17cf5aad1f20a",
-  "core-latest-tag": "v1.14.0-tm-v0.34.23",
-  "core-latest-sha": "ab4ef71b202dd9d27c879a820fa06e127d4234fa",
+  "app-latest-sha": "1943735bdefc751ae98732760921d49a3534a70a",
+  "core-latest-tag": "v1.15.0-tm-v0.34.23",
+  "core-latest-sha": "77a1de71feadebaa89c5ecaef0309fd8b9f2f921",
   "node-latest-tag": "v0.7.0",
-  "node-latest-sha": "7226f02794bdf11a91116d0d4cd88399f05149ad",
+  "node-latest-sha": "562016470d0f465d7c4f4d2bffcd44372c1172f5",
```

Comment:
Should be v0.7.1 now

Reply:
im closing this and re-running because 0.7.1 is latest